Misc/NEWS.d/next/Library/2019-03-04-16-39-16.bpo-36186.Hqw1A_.rst

```diff
@@ -0,0 +1 @@
+Fix handle leak in linuxaudiodev.
```

Comment:
"handle leak": it's more a file descriptor than a handle, no? I suggest:

Fix linuxaudiodev.linux_audio_device() error handling: close the internal file descriptor if it fails to open the device.